frontend/src/pages/TutorialPage/index.tsx

```diff
@@ -0,0 +1,187 @@
+            here
+          </Link>{" "}
+          which contains a model prediction for the sst2 dataset.
+          <ol>
+            <li>
+              On the side menu on the right, click on the{" "}
+              <Text strong>Systems</Text> tab.
+            </li>
+            <Image
+              src={"/screenshots/systemsTab.png"}
+              preview={false}
+              style={{
+                width: "200px",
+                maxWidth: "90%",
+                margin: "10px 0 10px 100px",
+              }}
+            />
+            <li>
+              Along the filter bar in the systems page, click the{" "}
+              <Text strong>New</Text> button.
+            </li>
+            <Image
+              src={"/screenshots/filterBar.png"}
+              preview={false}
+              style={{ maxWidth: "90%", marginLeft: "50px" }}
+            />
+            <li>
+              For <Text strong>Task</Text>, select
+              &quot;text-classification&quot;.
+            </li>
+            <li>
+              In the <Text strong>Dataset</Text> box, search for{" "}
+              <Text code>sst2</Text> and click on it.
+            </li>
+            <li>
+              In the dataset <Text strong>Split</Text> box, select
+              &quot;test&quot;. Meaning this is the model prediction for sst2
+              test split.
+            </li>
+            <li>
+              Click on{" "}
+              <Text strong>
+                <PlusOutlined />
+                Add File
+              </Text>{" "}
+              and choose the <Text code>output_sst2.txt</Text> file you just
+              downloaded.
+            </li>
+            <li>
+              In <Text strong>System Name</Text>, enter any name you want, but
+              you cannot leave it blank.
+            </li>
+            <li>
+              In <Text strong>Metrics</Text>, select &quot;Accuracy&quot;.
+            </li>
+            <li>
+              You should have something that looks like this in the end. Click
+              on the <Text strong>Submit</Text> button to submit.
+            </li>
+            <Image
+              src={"/screenshots/submissionDrawer.png"}
+              preview={false}
+              style={{
+                width: "700px",
+                maxWidth: "90%",
+                margin: "10px 0 10px 50px",
+              }}
+            />
+            <li>
+              After you submit, you should be able to find your system on the
+              systems page!
+            </li>
+            <Image
+              src={"/screenshots/submitSuccess.png"}
+              preview={false}
+              style={{ maxWidth: "90%", margin: "10px 0 10px 50px" }}
+            />
+          </ol>
+        </Paragraph>
+        <Divider />
+        <Title level={2} id="view-analysis">
+          View detailed analysis of your outputs
```

Comment:
I found the examples here a bit confusing, as they were not from the SST dataset and were using many different evaluation metrics. It'd probably be best to keep the examples consistent.